src/GeoJSON.jl

```diff
@@ -2,20 +2,121 @@ __precompile__()
 
 module GeoJSON
 
-    using JSON, Compat
-
-    export
-        Point,
-        MultiPoint,
-        LineString,
-        MultiLineString,
-        Polygon,
-        MultiPolygon,
-        GeometryCollection,
-        Feature,
-        FeatureCollection,
-        geojson
-
-    include("types.jl")
-    include("parser.jl")
+    import GeoInterface
+    import JSON
+
+    export parse,
+           parsefile,
```

Comment:
`parse` is a Base function that should probably not be exported. I see that both `parse` and `parsefile` exist in JSON but both not exported.